tests/test_products.py

```diff
@@ -0,0 +1,989 @@
+import unittest
+import numpy as np
+# import copy
+
+# from affine import Affine
+# import telluric as tl
+# from telluric.constants import WEB_MERCATOR_CRS
+from telluric.products import ProductError
+
+from common_for_tests import (
+    make_test_raster, multi_raster_8b, multi_values_8b, expected_nir, expected_red, expected_green, sensor_bands_info,
+    expected_blue, nir_value, red_value, green_value, blue_value, hyper_raster, hyper_raster_with_no_data,
+    multi_raster_with_no_data, hyper_bands,
+)
+
+from telluric.products import (
+    ProductsFactory, ProductGenerator, NDVI, EVI2, ENDVI, EXG, EXB, EXR, TrueColor, PRI, NDVI827, NRI, GNDVI, CCI,
+    NPCI, PPR, NDVI750, LandCoverIndex
+)
+
+
+class TestProductsFuctory(unittest.TestCase):
```

Comment:
Could we do some spell checking before merging this?

Reply:
I should :)